tests/test_modeling_common.py

```diff
@@ -3260,9 +3260,7 @@ def test_flash_attn_2_inference(self):
                 )
                 model_fa.to(torch_device)
 
-                model = model_class.from_pretrained(
-                    tmpdirname, torch_dtype=torch.bfloat16, attn_implementation="flash_attention_2"
-                )
+                model = model_class.from_pretrained(tmpdirname, torch_dtype=torch.bfloat16)
```

Comment:
let's update the name to `test_flash_attn_2_inference_equivalence` or something like that!

Reply:
You'll need to run the tests manually. You can select just the flash attention tests by doing something like: 

`RUN_SLOW=1 pytest tests/models -k "flash_attn"` on a GPU setup